callback.php

```diff
@@ -89,6 +89,14 @@
                 if ($order[0]['bits'] > $order[0]['bits_payed']) {
                     $o->setCurrentState(Configuration::get('PS_OS_ERROR'));
                 } else {
+                    $id_currency = Db::getInstance()->ExecuteS(
```

Comment:
We have already collected the id_currency along with the PrestaShop order $o. Simply using `$o->id_currency` should produce the same value as collecting it from the database again `$id_currency[0]["id_currency"]`

Reply:
You are right, I changed it and tried it. Works excellent. Thanks!